pkg/stanza/fileconsumer/internal/fingerprint/fingerprint.go

```diff
@@ -18,20 +19,42 @@ const MinSize = 16 // bytes
 // Fingerprint is used to identify a file
 // A file's fingerprint is the first N bytes of the file
 type Fingerprint struct {
-	FirstBytes []byte
+	FirstBytes   []byte
+	HashBytes    uint64
```

Comment:
Similarly, do we benefit from saving this vs just calling `HashInstance.Sum64()`?